### DISCUSSION TITLE
Hooks in case of hosting Pocketbase with Elestio

### DISCUSSION THREAD

**Discussion by jeansebastienvincent** [pocketbase/pocketbase#3824] [Q&A]
I have hosted Pocketbase with Elestio (https://elest.io).
This works fine but I do not know where I should add the pb_hooks folder.
Elestio offers a File Explorer tool but the structure is different then in a local setup.
So question is : where do we create the hooks folder ?
![image](https://github.com/pocketbase/pocketbase/assets/62613889/99ebefac-0421-4068-8514-7e1d5f633859)

**Comment by ganigeorgiev** [comment]
If you haven't explicitly set the `--hooksDir` flag when starting the binary, then by default the `pb_hooks` location is expected to be next to your `pb_data`.

Where that would be in the case of Elestio unfortunately I'm not sure as I cannot find in their documentation how it is deployed. You can try to use the Search bar and look for "pocketbase", "pb_data", "pb_hooks", "pb", etc.
